Sources/Controllers/2.0/Settings/NavigationSettings/OAVoiceAnouncesViewController.h

```diff
@@ -1,20 +0,0 @@
-//
```

Comment:
There is no "*.m" file for this hasher. Also, this file is not used in any other project file. It looks like a mistake. It looks like it was created by mistake.